src/index.html

```diff
@@ -6,7 +6,442 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="mainLogo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a href="#notebooks"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadjets" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="container">
```

Comment:
```suggestion
      <div class="container">
```
Why do you need additional wrapper